docs/components/concepts/process-instance-migration.md

```diff
@@ -90,6 +90,28 @@ Simply cancel the service task instance, and add a new instance of the service t
 ![The process instance can be modified to recreate the service task's job.](assets/process-instance-migration/migration-active_after-modification.png)
 :::
 
+## Correcting mistakes in a process instance
+
+Process instance migration can also be used to correct mistakes that might lead to an incident in a process instance.
+
+Let's consider an example.
+
+A user deployed a process with a user task `A` and input mappings. A process instance gets stuck at this user task because of a mistake (e.g. a typo) in the input mappings,
+and an incident is created to inform the user of this problem.
+
+![After creating the process instance, an incident is created for the service task A.](assets/process-instance-migration/migration-process_instance_with_incident.png)
```

Comment:
I like that idea. Do you see any common examples that could be resolved by migrating to a new process definition?

Reply:
I am not sure which ones are common tbh. What about referencing to the [incident concept page](https://docs.camunda.io/docs/next/components/concepts/incidents/)? Or we can directly use some examples from there. E.g.:

- A condition is not setup correctly so that it doesn't return `true` or `false`.
- Wrong variable used for decision evaluation resulting in an incident.

Reply:
I like the example with the condition. Consider a process instance that always receives two variables (one boolean and one string). By accident, the string is used in the condition. Rather than adjust the variable (which is not what you want to do), you need to adjust the process model.

It's a lot like a typo in an input mapping, but I guess that's almost always the case.

Reply:
@korthout I sent updates, please take a look. 👀